docs/playground/README.md

```diff
@@ -0,0 +1,90 @@
+# Plutus Playgrounds
+
+*Plutus Playgrounds* is a lightweight, easy-to-use, web-based environment for exploratory Plutus development. It facilitates the development and execution of Plutus contracts without the overhead of installing and maintaining a full development environment and blockchain testnet. Nevertheless, it is built around the original Plutus platform components, including the use of Plutus Core for on-chain contract code, and supports the full Plutus language. Hence, contracts developed with Plutus Playgrounds can eventually be deployed to the Cardano blockchain.
+
+The current version of this document sets out the minimal expectations for the v1.0 launch in addition with clearly labelled optional (nice to have, but not critical) functionality for v1.0.
+
+## User Interface
+
+The user interface has three main panes: (1) the *editor pane,* (2) the *wallet pane,* and (3) the *mockchain pane.*
```

Comment:
s/chain/mockchain/ throughout. From the user's point of view the fact that it's an emulator doesn't matter - it represents the chain.